packages/perseus/src/widgets.ts

```diff
@@ -176,9 +181,8 @@ export const getPublicWidgets = (): ReadonlyArray<WidgetExports> => {
     // @ts-expect-error - TS2740 - Type 'Pick<{ [key: string]: Readonly<{ name: string; displayName: string; getWidget?: (() => ComponentType<any>) | undefined; accessible?: boolean | ((props: any) => boolean) | undefined; hidden?: boolean | undefined; ... 10 more ...; widget: ComponentType<...>; }>; }, string>' is missing the following properties from type 'readonly Readonly<{ name: string; displayName: string; getWidget?: (() => ComponentType<any>) | undefined; accessible?: boolean | ((props: any) => boolean) | undefined; hidden?: boolean | undefined; ... 10 more ...; widget: ComponentType<...>; }>[]': length, concat, join, slice, and 18 more.
     return _.pick(
         widgets,
-        // @ts-expect-error - TS2345 - Argument of type '(name: string) => boolean | undefined' is not assignable to parameter of type 'Iteratee<string[], boolean, string>'.
         _.reject(_.keys(widgets), function (name) {
-            return widgets[name].hidden;
+            return widgets[name].hidden || name === "input-number";
         }),
```

Comment:
We don't want to register the input number widget any more, in order to keep it out of the list of available widgets while still allowing us to replace the widget with numeric-input.